static/js/index.js

```diff
@@ -219,7 +219,7 @@
 
     if (result.ResultCount) {
```

Comment:
Thanks! Updated.

On Tue, Mar 15, 2016 at 10:01 PM, Sylvain Zimmer <notifications@github.com>
wrote:

> In static/js/index.js
> <https://github.com/commonsearch/cosr-front/pull/29#discussion_r56282310>:
>
> > @@ -219,7 +219,7 @@
> >
> >      if (result.ResultCount) {
>
> should patch here too?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/commonsearch/cosr-front/pull/29/files/42ffa509ec792b1970e47adaa04a209f3d31208c..ed20e173a7e1c342f4e8472cd8e050368bef25a3#r56282310>
>



-- 
Yu Wu